File: Books Price Python/utils/mongodb_connection.py
from pymongo import MongoClient

class MongoDBConnection:
    _database = None

    def __init__(self):
        pass  

    @staticmethod
    def get_database():
        if MongoDBConnection._database is None:
            client = MongoClient(
                "mongodb+srv://Arelis:Arelis2006@cluster0.qdn4zsf.mongodb.net/"
            )
            MongoDBConnection._database = client["SalesDB"]

        return MongoDBConnection._database

File: BookSaleSystem/src/main/java/ec/edu/espe/booksalesystem/utils/MongoDBConnection.java
package ec.edu.espe.booksalesystem.utils;

import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoDatabase;

/**
 *
 * @author Arelis Samantha Bonilla Cruz, Student, @ESPE
 */
public class MongoDBConnection {
    private static MongoDatabase database;

    private MongoDBConnection() {
    }

    public static MongoDatabase getDatabase() {
        if (database == null) {
            MongoClient client = MongoClients.create("mongodb+srv://Arelis:Arelis2006@cluster0.qdn4zsf.mongodb.net/");
            database = client.getDatabase("BookSalesDB");
        }
        return database;
    }
}